frontend/src/components/HospitalInventoryEdit.module.css

```diff
@@ -0,0 +1,14 @@
+.inventoryInput {
+  /* width: 40%!important; */
+  text-align: center;
+}
+
+.inputCard {
+  border: none!important;
+  /* width: 160px; */
+  margin: 0 2px;
+}
+
+.containerDiv {
+  margin: 0 10%;
```

Comment:
Note: alternatively, you can also use the `my-*` classes from Bootstrap to set vertical margins (see https://getbootstrap.com/docs/4.0/utilities/spacing/ for details).